data/campaigns/Descent_Into_Darkness/scenarios/02_Peaceful_Valley.cfg

```diff
@@ -379,17 +401,17 @@
 
         [message]
             speaker=Malin Keshar
-            message= _ "Sacking goblin villages is all well and good, but walking corpses and bats won’t stand up to full orcish warriors."
+            message= _ "Sacking goblin villages is all well and good, but walking corpses and bats cannot stand up to full orcish warriors."
         [/message]
 
         [message]
             speaker=Darken Volk
-            message= _ "Your training is just beginning, Malin. With the souls from these goblins now available for our use, you will see your forces grow quickly."
+            message= _ "Have some patience. We have only just begun your... training. These corpses and souls will supply you sufficient resources for practice. When you have mastered the basics, we will then assault the orcs and you will see your forces grow quickly."
         [/message]
 
         [message]
             speaker=Malin Keshar
-            message= _ "Excellent. I’m eager to move on to the real foes."
+            message= _ "Very well."
```

Comment:
I think at least "I'm eager to move on to the real foes" sounds better.

Reply:
I didn't think it made sense because from the story text, the bulk of their fighting the orcs occurs around S4-5, which is a little bit later than here. The original is better flavor text, I think, I just wasn't sure if it was applicable. (Might revert to original, anyone else have any opinions?)

Reply:
How about being less definite: "I am eager to move on to some real foes."

Reply:
Done.